agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -1,113 +1,117 @@
-#script to log and readout pfeiffer TPG 366 gauge contoller
-#via Ethernet connection
-#Zhilei Xu, Tanay Bhandarkar
+# Script to log and readout pfeiffer TPG 366 gauge contoller
+# via Ethernet connection
+# Zhilei Xu, Tanay Bhandarkar
 
 import socket
 import numpy as np
-from ocs import ocs_agent, site_config, client_t
+from ocs import ocs_agent, site_config
 from ocs.ocs_twisted import TimeoutLock
 import time
 
 BUFF_SIZE = 128
 ENQ = '\x05'
-IP_ADDRESS = '10.10.10.20'
+
 
 class pfeiffer:
```

Comment:
PEP08 recommendation is classes use CapWords format. Capitalize pfeiffer.

https://www.python.org/dev/peps/pep-0008/#class-names